config/README.rst

```diff
@@ -203,7 +203,7 @@ updated. Example:
         "# E222 multiple spaces after operator",
         "per-file-ignores =",
         "    src/foo/bar.py: E221 E222",
-        "ignore = D203",
+        "ignore = D203, W503",
```

Comment:
Let's show a good example by commenting what these error codes mean?

Also, I think for flake8 it's better to specify `extend-ignore =` rather than `ignore =`.

Reply:
TIL + fixed in https://github.com/zopefoundation/meta/pull/61/commits/293bd3ddf5540e1896be713f81baa2c1b367b5dc